### PR TITLE
[WasteWorks] Update phone numbers on payment failed emails

### PR DESCRIPTION
**Pull request by chrismytton**
For FD-3619

### CODE REVIEW

**Review by dracos — CHANGES_REQUESTED** [review]
couple of queries about the new text

File: templates/email/default/waste/csc_payment_failed.html [hunk 1]
```diff
@@ -13,7 +13,15 @@
   <h1 style="[% h1_style %]">There was a problem with your payment</h1>
   [% IF report.category == 'Bulky collection' %]
     <p style="[% p_style %]">There was a problem taking payment for your [% report.title %].</p>
+    [% IF c.cobrand.moniker == 'peterborough' -%]
     <p style="[% p_style %]">Please contact 01733 747474; if you feel you will have difficulty with the auto payment system please advise the call handler at the start of the call.</p>
+    [% ELSIF c.cobrand.moniker == 'bromley' -%]
+    <p style="[% p_style %]">Please contact 0300 303 8658; if you feel you will have difficulty with the auto payment system please advise the call handler at the start of the call.</p>
+    [% ELSIF c.cobrand.moniker == 'sutton' -%]
+    <p style="[% p_style %]">Please contact 020 8770 5000; if you feel you will have difficulty with the auto payment system please advise the call handler at the start of the call.</p>
+    [% ELSE -%]
```
[the remark this answers is not in the document]
Comment (chrismytton) [reply templates/email/default/waste/csc_payment_failed.html:22]: From what I could tell the `csc_payment_failed` method only runs if `$c->stash->{staff_payments_allowed} eq 'paye'`, but looks like Kingston is set to `cnp`

https://github.com/mysociety/fixmystreet/blob/f3eb2477a73218e34d131b1162bbcbdb9f321641/perllib/FixMyStreet/App/Controller/Waste.pm#L542-L544

https://github.com/mysociety/fixmystreet/blob/f3eb2477a73218e34d131b1162bbcbdb9f321641/perllib/FixMyStreet/Cobrand/Kingston.pm#L48

File: templates/email/default/waste/csc_payment_failed.html [hunk 2]
```diff
@@ -13,7 +13,15 @@
   <h1 style="[% h1_style %]">There was a problem with your payment</h1>
   [% IF report.category == 'Bulky collection' %]
     <p style="[% p_style %]">There was a problem taking payment for your [% report.title %].</p>
+    [% IF c.cobrand.moniker == 'peterborough' -%]
     <p style="[% p_style %]">Please contact 01733 747474; if you feel you will have difficulty with the auto payment system please advise the call handler at the start of the call.</p>
+    [% ELSIF c.cobrand.moniker == 'bromley' -%]
+    <p style="[% p_style %]">Please contact 0300 303 8658; if you feel you will have difficulty with the auto payment system please advise the call handler at the start of the call.</p>
```
[the remark this answers is not in the document]
Comment (chrismytton) [reply templates/email/default/waste/csc_payment_failed.html]: Ah yes, just copied that without thinking! Changed in edefa42deb68f2212136f7c6021d7021bac1e408

**Review by dracos — APPROVED** [review]
Woo 🖲️

**Pull request merged**